inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/random_uniform/random_uniform_kernel_ref.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+
+#include "kernel_base_opencl.h"
+
+namespace kernel_selector {
+
+/**
+ * Random Uniform kernel params. All the needed inputs are in base input, output shape is static and
+ * presented in output.
+ */
+struct random_uniform_params: public base_params {
+    random_uniform_params() :
+        base_params(KernelType::RANDOM_UNIFORM) {
+    }
+
+    // operation attributes
+    uint64_t global_seed;
+    uint64_t op_seed;
+};
+
+/**
+ * Specific optional params is not defined for RandomUniform operation.
+ */
+struct random_uniform_optional_params: optional_params {
+    random_uniform_optional_params() :
+        optional_params(KernelType::RANDOM_UNIFORM) {
+    }
+};
+
+/**
+ * Reference GPU kernel for the RandomUniform-8 operation.
+ */
+class RandomUniformKernelRef: public KernelBaseOpenCL {
+public:
+    RandomUniformKernelRef() :
+        KernelBaseOpenCL("random_uniform_ref") {
+    }
+
+    KernelsData GetKernelsData(const Params &params, const optional_params &options) const override;
+
+    KernelsPriority GetKernelsPriority(const Params &params, const optional_params &options) const override;
+
+    ParamsKey GetSupportedKey() const override;
+
+    bool Validate(const Params &p, const optional_params &o) const override;
```

Comment:
Maybe params/options instead of p/o - to have consistent argument names for all the methods of the class?